### DISCUSSION TITLE
is impossible to add page break and make the editor into 2 pages when printing?

### DISCUSSION THREAD

**Discussion by crying2812** [ueberdosis/tiptap#888] [General Questions]
Hi guys, i', using Tiptap and want to add some features for it, i wonder is there any way when user add long text, the editor when auto add page break and separate them into 2 pages?

**Comment by hanspagel** [comment]
This is probably not really related to tiptap, but you can apply CSS to force page breaks for specific elements. Hope that helps!

https://developer.mozilla.org/de/docs/Web/CSS/page-break-after

**Reply by crying2812** [reply]
Thanks for reply, one more question related to it: How can i check the height of paragraph then add page-break to editor?

**Comment by aehlke** [comment]
Take a look at paged.js as well

**Comment by Sushant-summit** [comment]
Guys just create a custom node named page break and in render HTML, provide the page break styles. The node code is going to be same as HorizontalRule.
